docs/source/models/intervention_models/intrapartum/azithromycin_intervention.rst

```diff
@@ -0,0 +1,265 @@
+.. _azithromycin_intervention:
+
+=========================================
+Azithromycin for treating maternal sepsis
+=========================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. list-table:: Abbreviations
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+    - Note
+  * - BEmONC
+    - Basic emergency obstetric and neonatal care
+    - Operationalized as facilities without C-section capabilities
+  * - CEmONC
+    - Comprehensive emergency obstetric and neonatal care
+    - Operationalized as facilities with capabilities to perform  C-section
+  * - PAF
+    - Population Attributable Fraction
+    - 
+  * - RR
+    - Relative Risk
+    - 
+  * - SARA
+    - Services Availabiity and Readiness Assessment
+    - 
+
+Intervention Overview
+-----------------------
+
+Azithromycin is used to prevent maternal sepsis in the intrapartum period, reducing the risk incidence.
+
+This section describes how an azithromycin intervention can be implemented and calibrated for the :ref:`MNCNH Portfolio model <2024_concept_model_vivarium_mncnh_portfolio>`.
+
+.. list-table:: Affected Outcomes
+  :widths: 15 15 15 15
+  :header-rows: 1
+
+  * - Outcome
+    - Effect
+    - Modeled?
+    - Note (ex: is this relationship direct or mediated?)
+  * - Maternal sepsis and other maternal infections Incidence Probability :math:`\text{CSMR}_i^\text{sepsis}`
+    - Adjust multiplicatively using RR
+    - Yes
+    - For convenience, we will model this like a dichotomous risk factor; more details below
+
+Baseline Coverage Data
+++++++++++++++++++++++++
+
+Baseline coverage data for azithromycin in CEmONC and BEmONC is only reflective of Ethiopian health systems in 2015-2018. These 
+placeholder values come from three data sources, all for Ethiopia, all identified by the Health Systems team at IHME: the 2016 
+Ethiopia EmONC Final Report found 39.2% of BEmONC facilities and 90.4% of CEmONC facilities have azithromycin; the 2016 SARA Report 
+found 68.4% of BEmONC facilities and 90.5% of CEmONC facilities have azithromycin; the 2018 SARA Report found 75.9% of BEmONC 
+facilities and 98.0% of CEmONC facilities have azithromycin. While we plan a data strategy to fill the gaps we have used a simple 
+average.
+
+.. list-table:: Baseline Coverage of intrapartum azithromycin
+  :widths: 15 15 15 15
+  :header-rows: 1
+
+  * - Birth Facility
+    - Coverage Mean (%)
+    - Coverage Distribution (%)
+    - Notes
+  * - Home Birth
+    - 5
+    - :math:`\text{Uniform}(0,10)`
+    - Assumption; need to investigate data sources for care seeking those giving birth outside of the hospital system 
+  * - BEmONC Facilities
+    - 61.2
+    - :math:`\text{Uniform}(39.2,75.9)`
+    - placeholder value based on three data points 
+  * - CEmONC Facilities
+    - 93.0
+    - :math:`\text{Uniform}(90.4,98.0)`
+    - placeholder value based on three data points 
+
+
+Vivarium Modeling Strategy
+--------------------------
+
```

Comment:
How do we want to handle partial term pregnancies that are not assigned delivery facilities?